docs/general/electronics.md

```diff
@@ -64,6 +64,15 @@ To start you off, here are some miscellaneous tips you might need on your journe
 - Don't use silicone insulated wire in drag chains  
     The excess friction causes the drag chain to not work properly and wear your wires prematurely.
 
+---
+
+{: .warning }
+> Make sure you've set the input voltage selector switch on your PSU if applicable!
+>
+>- Selecting 220v in a 110v country will cause your PSU to run under its rated power.
```

Comment:
220v -> 220V
110v -> 110V

Probably we should mention PSU(s) since the kit comes with two 😀